recipes/hiproc/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "hiproc" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a25cbf8d054fe1caa6d69c5339ed4248ac733f34f69b83fe7c96f6bd3366e7c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.18.1
+    - scipy >=1.4.1
+    - matplotlib >=3.2.1
```

https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base >=3.2.1
```